doc/user/content/manage/dbt/_index.md

```diff
@@ -489,6 +489,53 @@ and also accessible in memory within the [cluster](/concepts/clusters/) the
 index is created in. Indexes help optimize query performance and make queries
 against materialized views faster.
 
+##### Using refresh strategies
+
+{{< tip >}}
+For guidance and best practices on how to use refresh strategies in Materialize,
+see [Refresh strategies](/sql/create-materialized-view/#refresh-strategies).
+{{</ tip >}}
+
+{{< private-preview />}}
+
+For data that doesn't require up-to-the-second freshness, or that can be
+accessed using different patterns to optimize for performance and cost
+(e.g., hot vs. cold data), you may consider using a non-default
+[refresh strategy](/sql/create-materialized-view/#refresh-strategies).
+
+To tweak the refresh strategy of a materialized view model, use the
+[`refresh_interval` configuration](#configuration-refresh-strategies). As a
+reminder, these models need to be deployed in a [scheduled cluster](/sql/create-cluster/#scheduling),
```

Comment:
Yeah. From the reference documentation:

> Because these views can be hosted in [scheduled clusters](https://materialize.com/docs/sql/create-cluster/#scheduling), which automatically turn on and off based on the configured refresh strategies, this feature can lead to significant cost savings when handling large volumes of historical data that is updated less frequently.

Adapted to make it a bit clearer here as well!